components/brave_new_tab_ui/components/default/braveToday/cards/displayAd/index.tsx

```diff
@@ -37,7 +37,7 @@ export default function CardDisplayAd (props: Props) {
     let onItemViewed = props.onViewedDisplayAd
     const observer = new VisibilityTimer(() => {
       onItemViewed({ ad: content })
-    }, 100, innerRef.current)
+    }, 1000, innerRef.current)
```

Comment:
should values like this be defined as constants?

Reply:
Agreed. I will raise a follow up issue

Reply:
Thanks